ui/tsconfig.json

```diff
@@ -20,6 +20,8 @@
       "vault/tests/*": ["tests/*"],
       "vault/mirage/*": ["mirage/*"],
       "vault/*": [
+        "types/*",
+        "types/vault/*",
```

Comment:
I noticed that type definitions in the `types` directory were not being resolved and adding these 2 paths fixed it.